data/constants/keycodes/extras/keycodes_canadian_french_0.0.1.hjson

```diff
@@ -0,0 +1,407 @@
+{
+    "aliases": {
+/*
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───────┐
+ * │ # │ 1 │ 2 │ 3 │ 4 │ 5 │ 6 │ 7 │ 8 │ 9 │ 0 │ - │ = │       │
```

Comment:
This does not quite match the French Canadian layout as the Windows OSK reports it:
![image](https://github.com/qmk/qmk_firmware/assets/4781841/4347c463-faac-442d-8c98-b341792e2b78)

Reply:
Which input locale did you use on Windows when reviewing this PR?

Here is a screenshot of what exact input locale this PR refers to:
<img width="341" alt="Canadian French input locale selection" src="https://github.com/qmk/qmk_firmware/assets/2391500/ddbf838e-261e-42f2-9b20-ab0df3ddb584">

And a screenshot of the "Canadian French" input locale chosen above in Windows 10's On-Screen Keyboard application:
<img width="758" alt="Canadian French input locale as shown on the On-Screen Keyboard, Windows 10" src="https://github.com/qmk/qmk_firmware/assets/2391500/6c649b61-cf6f-4d94-ac86-44c2227cbfd4">